packaging/debian/postinst

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 
 #DEBHELPER#
-mkdir -p /var/log/percona
-chmod -R go+w /var/log/percona
+mkdir -p /var/log/percona/telemetry-agent
```

Comment:
for the time being installations that already have TA installed have `/var/log/percona` directory with 777 permissions. Will we change it?

Reply:
I don't think we can do much about the packages that already have TA installed until they update the packages to the latest TA released with this change. @EvgeniyPatlan can provide more information on the plan to proceed with the release of the TA packages with this fix.